CMakeLists.txt

```diff
@@ -467,6 +468,7 @@ if (ISLE_BUILD_APP)
   add_executable(isle WIN32
     ISLE/res/isle.rc
     ISLE/isleapp.cpp
+    ISLE/isledebug.cpp
```

Comment:
Would it make sense to add a conditional/flag to either include or exclude this (together with IMGUI)?

Reply:
If you don't run with `--debug`, then you won't notice it.

Reply:
Yes, but IMGUI and isledebug would still be part of the executable, right? Seems like significant overhead (`ISLE.EXE` 1.8MB with debug, vs 98kb without on my win32 debug build)

Reply:
Oh, yes. Good point. I'll make it an option (default ON?) then.

Reply:
There is `ISLE_DEBUG` now, which defaults to `ON`.
Once we understand every bit, we can change this default ;)